### PR TITLE
{toolchain} GCC 11.4.0

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.2
See https://gist.github.com/SebastianAchilles/0fb68a4b7372af7123601e7b244114f0 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
skl-fedora-39 - Linux Fedora Linux 39 (Container Image Prerelease), x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.11.3
See https://gist.github.com/SebastianAchilles/c30f62d41db48d1b5894adb1bda6706c for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 525.105.17, Python 3.9.16
See https://gist.github.com/SebastianAchilles/9003c21b6ba1213cda7663679b462b97 for a full test report.

File: easybuild/easyconfigs/g/GCCcore/GCCcore-11.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    'gmp-6.2.1.tar.bz2',
+    'mpfr-4.2.0.tar.bz2',
+    'mpc-1.3.1.tar.gz',
+    'isl-0.26.tar.bz2',
+    'newlib-4.3.0.20230120.tar.gz',
+    {'download_filename': '93e0090.tar.gz', 'filename': 'nvptx-tools-20230122.tar.gz'},
+]
+patches = [
+    'GCCcore-6.2.0-fix-find-isl.patch',
+    'GCCcore-9.3.0_gmp-c99.patch',
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-11.4.0.eb:35]: @SebastianAchilles I have two extra patches in my PR #17993, those still seem relevant?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
closing in favour of #17993

**Pull request closed without merging**